_posts/2017-11-07-haskell-learning-group-our-progress-and-updates.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Haskell Learning Group: Our Progress and Updates"
+date: 2017-11-07 00:03:30 +0200
+categories: haskell how-we-learn
+author_name: Sergii Paryzhskyi
+author_url : /author/sergii_paryzhskyi
+author_avatar: sergii_paryzhskyi
+show_avatar : true
+read_time : 7
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-17-first-haskell-meeting/haskell-poster.jpg
+---
+
+This post is a quick update and the latest news about our Haskell learning group. Even though there were no blog posts on this topic recently, we are still having weekly meetings and mostly stick to the plan. That of course implies preparation time, spend between our sessions. There is no rush in our learning process and we take time to discuss interesting topics a bit longer, especially when all participants are involved. That being said, we might have a plan to go through one chapter per session but it also fine if it is less than that.
+
+As expected the difficulty is slowly increasing as we go along. We are spending more time reading chapters. Sometimes the same material has to be read multiple times in order to fully understand what is it about.
+
+It helps a lot that the most of the concepts are familiar to us through other languages or libraries we are using. For instance JavaScript doesn't support currying natively the way Haskell does it, but there are libraries which introduce these concepts. We are very often using [RamdaJS][ramdajs] in our projects, so such things as function composition, currying, partial application and so on were not something totally new to us. They still had to be "translated" and demonstrated in a Haskell way but we certainly didn't need much time to get the idea.
```

Comment:
> doesn't support currying natively the way Haskell does it

JS doesn't have any kind of currying features. Not "natively" (not entirely sure the use of this word actually makes sense here?) or otherwise and not like Haskell either. Perhaps you could just rephrase it to something more like: 

"While Javascript functions are not curried, currying can can be simulated: libraries like Ramda or Lodash provide `curry` functions providing this functionality"

Also you are talking only about currying but then refer to currying as "these concepts" and only later introduce "function composition, partial application and so on"

Reply:
"natively" -> "doesn't curry function arguments by default".

I see the problem here. I will rewrite it in another way